### ISSUE TITLE
<jquery-atmosphere>Cross domain requests with IE not working

### ISSUE THREAD

**Issue by djotanov** [Atmosphere/atmosphere#425]
Cross domain requests with IE8 are not working. I'm using long-polling as transport protocol. The client connection is correctly suspended and client can receive broadcasted messages. It is just sending part that does not work.
I'm getting:
 SCRIPT70: Permission denied jquery.atmosphere.js, line 1139 character 21

By looking through jquery-atmosphere.js, it seems that regular ajax call is sent to server when sending message (where it should be XDR ajax request or jsonp?).

I have a quick and dirty fix to take into consideration:

``` js
function _pushIE(message) {
                if (_request.enableXDR) {
                    var rq = _request;
                    rq.attachHeadersAsQueryString = true;
                    var url = _attachHeaders(rq);
                    url += "&X-Atmosphere-Post-Body=" + encodeURIComponent(message);
                    jQuery.ajax({
                        url : url,
                        type : "POST",
                        contentType : "application/json;charset=UTF-8",
                        scriptCharset: "UTF-8",
                        dataType: "jsonp",
                        jsonp : "jsonpTransport"
                    });
                } else 
                    _pushAjaxMessage(message);
}
```

Basically, jsonp request is sent for every push from IE when enableXDR is true.
